ldk/javascript/src/index.ts

```diff
@@ -1,3 +1,4 @@
+import "core-js";
```

Comment:
How is this being used if we are not exporting it?

Reply:
This one is polyfilling and modifying the global scope